core/src/test/java/org/elasticsearch/threadpool/SimpleThreadPoolIT.java

```diff
@@ -115,7 +115,7 @@ public void testThreadNames() throws Exception {
         for (String threadName : threadNames) {
             // ignore some shared threads we know that are created within the same VM, like the shared discovery one
             // or the ones that are occasionally come up from ESSingleNodeTestCase
-            if (threadName.contains("[" + ESSingleNodeTestCase.nodeName() + "]")
+            if (threadName.contains("[node_s_0]") // TODO: this can't possibly be right! single node and integ test are unrelated!
```

Comment:
this seems unintuitive but I think was correct. If we start a single node test the threads are always around since we don't tear it down?